### PR TITLE
hotfix - download refts res via rest api

### PR DESCRIPTION
**Pull request by zhiyuli**
a hotfix for #1363

### CODE REVIEW

**Comment by zhiyuli** [comment]
@pkdash could you review this? thanks

File: ref_ts/views.py [hunk 1]
```diff
@@ -208,6 +209,7 @@ def create_ref_time_series(request, *args, **kwargs):
         context = {'resource_creation_error': "Error: failed to create resource." }
         return render_to_response('pages/create-ref-time-series.html', context, context_instance=RequestContext(request))
 
+@api_view(['GET'])
```
Comment (pkdash) [inline ref_ts/views.py]: Did you test if the view function works with the added decorator when download is initiated from the resource landing page?

**Comment by zhiyuli** [comment]
@pkdash yes, it still works on landing page. 
I shared a private refts res with you on park if you would like to test it.
https://park.hydroshare.org/resource/ce943b09e93e4dea84d3e3d521c538fc/

thanks

**Comment by pkdash** [comment]
+1

**Comment by zhiyuli** [comment]
@mjstealey 
Could you please apply this hotfix to www? thanks

**Pull request merged**